### PR TITLE
DOC-6815 guided install troubleshooting

### PR DESCRIPTION
**Pull request by paperclypse**
Documenting a couple of the most common issues with the guided install process. I'd love a developmental edit chat on this one, because I'm not sure this is the best place and way of documenting this.

### CODE REVIEW

File: src/content/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview.mdx [hunk 1]
```diff
@@ -173,4 +173,53 @@ Our open source integrations send performance metrics and inventory data from yo
 			<td>newrelic install -n varnish-cache-open-source-integration</td>
 		</tr>
 	</tbody>
-</table>
\ No newline at end of file
+</table>
+
+## Troubleshoot common problems
```
Comment (urbiz-nr) [inline src/content/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview.mdx:178]: header anchor?

File: src/content/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview.mdx [hunk 2]
```diff
@@ -173,4 +173,53 @@ Our open source integrations send performance metrics and inventory data from yo
...
+
+Guided install is fairly technically complex. As we identify areas where the guided install fails, we'll document them here and provide some guidance.
```
Comment (urbiz-nr) [inline src/content/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview.mdx:180]: "fairly technically" sounds a little weird.

File: src/content/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview.mdx [hunk 3]
```diff
@@ -173,4 +173,53 @@ Our open source integrations send performance metrics and inventory data from yo
...
+
+### MySQL: Incorrect user permissions
```
Comment (urbiz-nr) [inline src/content/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview.mdx:182]: Maybe we could use a collapser here for each section (MySQL and NGINX).

**Review by urbiz-nr — APPROVED** [review]

**Comment by urbiz-nr** [comment]
@paperclypse - I've left a couple of comments. You're good to go!

File: src/content/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview.mdx (hunk 3 above)
Comment (paperclypse) [reply src/content/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview.mdx:182]: Maybe!

**Comment by paperclypse** [comment]
I'm merging this in, because I've gotten an approval. Just had a merge conflict to resolve.

File: src/content/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview.mdx [hunk 4]
```diff
@@ -173,4 +173,70 @@ Our open source integrations send performance metrics and inventory data from yo
...
+As we identify areas where the guided install fails, we'll document them here and provide some troubleshooting guidance.
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="mysql"
+    title="MySQL: Incorrect user permissions"
+  >
+
+To monitor MySQL health data, you need a valid username and password for a user with specific permissions.
```
Comment (urbiz-nr) [inline src/content/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview.mdx:189]: I'd remove "for a user", it's a bit redundant.

File: src/content/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview.mdx [hunk 5]
```diff
@@ -173,4 +173,70 @@ Our open source integrations send performance metrics and inventory data from yo
...
+`status_url`: The URL set up to provide the metrics using the status module. If the default value of `127.0.0.1` is incorrect, substitute the address/FQDN/URL for your system. Example:
+
+    `status_url: http://127.0.0.1/status`
+
+You can read more about the `status_url` in these NGINX docs:
+
+* For NGINX Open Source: [HTTP stub status module](https://nginx.org/en/docs/http/ngx_http_stub_status_module.html)
+* For NGINX Plus: [HTTP status module](https://nginx.org/en/docs/http/ngx_http_status_module.html) and [HTTP API module](http://nginx.org/en/docs/http/ngx_http_api_module.html)
+
+There are a different ways to set `status_url`, depending on how NGINX was installed:
```
Comment (urbiz-nr) [inline src/content/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview.mdx:232]: Typo! Remove "a"

File: src/content/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview.mdx [hunk 6]
```diff
@@ -173,4 +173,70 @@ Our open source integrations send performance metrics and inventory data from yo
...
+
+* If enabled via Kubernetes: see [Monitor services running on Kubernetes](/docs/monitor-service-running-kubernetes).
```
Comment (urbiz-nr) [inline src/content/docs/full-stack-observability/observe-everything/get-started/new-relic-guided-install-overview.mdx:234]: We usually capitalize after :

File: src/content/docs/style-guide/article-templates/troubleshooting-docs-guide.mdx [hunk 7]
```diff
@@ -8,12 +8,7 @@ redirects:
   - /docs/new-relic-only/advanced-style-guide/article-templates/troubleshooting-docs-guide
 ---
 
-To create a new troubleshooting doc, use the troubleshooting doc template:
-
-1. From your New Relic Okta login, go to the Docs site: **[https://docs.newrelic.com/](https://docs.newrelic.com/)**.
-2. From the Drupal menu bar, select **My workbench > Create content > [Troubleshooting doc](https://docs.newrelic.com/node/add/troubleshooting-doc)**.
-
-This guide provides additional details for how to complete the troubleshooting doc template. For an example doc that uses all three standard troubleshooting sections, see [Resolve .NET and SCOM conflicts](/docs/agents/net-agent/troubleshooting/resolve-net-scom-conflicts).
+Our troubleshooting docs has these main sections: **problem**, **solution**, and (optionally) **cause**. Otherwise, a troubleshooting doc uses the [basic doc template](/docs/style-guide/article-templates/basic-doc-template)
```
Comment (urbiz-nr) [inline src/content/docs/style-guide/article-templates/troubleshooting-docs-guide.mdx:11]: Our troubleshooting docs HAVE

**Review by urbiz-nr — APPROVED** [review]

**Comment by paperclypse** [comment]
Merging, because I have gotten a couple of reviews on this.

**Pull request merged**